02week/ticTacToe.js

```diff
@@ -13,7 +13,7 @@ let board = [
 ];
 
```

Comment:
@LuisCarter8791 
You need to complete at least one of the requirements to receive credit for tictactoe, since the goal here was to pass the tests, unless your code passes at least one of the tests, then I can't give you any credit.